.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,28 +1,18 @@
 ---
-name: Feature issue template
-about: A broad template for feature-building issues
+name: Issue Template
+about: Each issue should have at least two labels (one to indicate the corresponding epic and one to indicate the nature of the ticket, e.g. client-side or server-side). GitHub milestones correspond to features of the product and issues will be grouped by the feature they relate to.
 
 ---
 
-# Issue Title
+**User Story:**
 
-Should be clear, precise and (where possible) written in non-technical language.
+**Product requirements/Acceptance criteria:**
 
-# Labels
-
-Each ticket should have at least two labels- one to indicate the corresponding epic and one to indicate the nature of the ticket.
-
-# Description
-
-**Plain text description**
-
-What are you building? What value is this technical task adding? Include relevant links/references
-
-**Technical description (optional)**
+**Technical description (optional):**
 
 Include relevant links/references
 
-**Possible solutions**
+**Possible solutions:**
```

Comment:
If there are many possible solutions, that would indicate to me that a discussion issue is required & presumably then an ADR. Or, if no discussion is needed, then requiring a dev to outline all the ways they _might_ implement something may not be especially illuminating?

So, if "Possible solutions" is filled out by someone, then the issue probably isn't ready to be picked up yet. With that in mind, is this a useful section to have in our template? Aren't we better off raising a new issue, which outlines the possible choices & prompts that discussion?

Genuine question - it might be that I can't think of a use case, but that there is one? :woman_shrugging: